.dockerignore

```diff
@@ -11,4 +11,7 @@ scripts/
 tilt-settings.json
 tilt.d/
 Tiltfile
+**/.tiltbuild
+test/infrastructure/docker/e2e/logs/**
+**/config/**/*.yaml
```

Comment:
I wouldn't think we need anything under the config directory, or is this to prevent accidental inclusion of other directories that might be named `config`?

Reply:
Hmm, maybe deserves a comment – the trouble is that setting the image pull strategy + arch modifies files under each manager's `config` directory. 

So, when we do the next `COPY . .` it's a different tree, and the `go build .` runs again, once per manager.

Reply:
Yeah, the point I was attempting to make is that I don't think we need the config directory at all in the docker images, since it only contains generated yaml and not code to be compiled/run in the container.

Reply:
Exactly – it's not needed by any of the images, but it does bust the cache on each image build. Seemed like a worthy `.dockerignore` entry :)